### DISCUSSION TITLE
Dedicar-se ao que a faculdade pede ou ao que de fato tenho predisposição?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1921] [Discussão]
Olá gente! Vou tentar ser o mais direta possível, visto que provavelmente muitos já tiveram essa dúvida em suas mentes.

Atualmente me vejo cursando TSI, um curso que apesar de ser focado em desenvolvimento de sistemas para web, tem muito mais enfoque no Back End, chega a ser ridícula a quantidade de disciplinas voltadas para Front End, sem contar que não há disciplinas sobre UX ou nada próximo a isso. Sempre fui mais para o lado da experiência do que da lógica, e agora, quase no final do curso, me vejo em meio a uma questão, tentar aprender, mesmo sendo péssima e sem nenhuma predisposição, o Back End, ou me especializar nas tecnologias e conceitos do Front End, sabendo que me vejo capaz de fazer isso?

**P.S.** Me sinto forçada pelo curso, e incapaz de certo modo. Sei que provavelmente irei conclui-lo, mas não me vejo em momento algum desenvolvendo para esse nicho. Penso em me especializar o máximo que puder em tecnologias aliadas as CSS, e melhorar minhas skills para ser boa com Front End e UX.

**Como vocês lidaram com isso? Foi difícil para vocês reconhecer que não eram bons em determinado aspecto? E quanto ao reconhecimento das pessoas à sua volta pela sua escolha?**

**Comment by lddv** [comment]
> quase no final do curso

Isso pra mim é um ótimo indicativo de persistir. Os conhecimentos adquiridos sobre o back-end não serão perdidos e poderão ajudar a compreender melhor aplicações com as quais você lidará no futuro.

Além do mais, mesmo que o seu foco seja experiência de usuário, o problema do usuário pode ser causado por uma má distribuição das requests do back-end. Digamos que existem muitos serviços sendo chamados e seja possível paralelizar isso. Esse tipo de oportunidade pode ser inicialmente percebida por uma ponta e resolver na outra ponta.

Espero que consiga ver que o seu interesse não precisa limitar o seu conhecimento em outras áreas afins e que podem completar em te fazer uma profissional mais completa.

>Como vocês lidaram com isso?

Ainda tenho muitas sub-áreas de interesse, como Ciência de Dados. Ouço podcasts e tento ficar inteirado sobre o que está acontecendo na área por alto. Mas como sou Front-end, tenho que focar no que preciso ter na ponta da língua/dedos 😅.

>Foi difícil para vocês reconhecer que não eram bons em determinado aspecto?

No início, você quer abraçar o mundo. Dê tempo ao tempo e foque no que te agrada mais, sem descartar os conhecimentos afins que podem agregar na sua área de interesse.

>E quanto ao reconhecimento das pessoas à sua volta pela sua escolha?

Essa luta, na minha opinião é muito mais com você mesma do que com quem tá de fora. E se você mudar de opinião daqui uns anos, tá tudo bem. Como eu disse antes, você sempre vai ter a sua bagagem e vai trazer os conhecimentos pra área que você quiser atuar.

Espero ter ajudado.

**Comment by ghost** [comment]
Estou tentando exatamente dá tempo ao tempo agora, focar com calma em cada coisa (sou introvertida e isso me faz ter uma mente um tanto acelerada 😅, o que de certo modo tem me atrapalhado bastante). Foco realmente é a chave, e estou me esforçando para consegui-lo @lddv. Ajudou demais, obrigada e boa sorte para nós! 😉

**Comment by renedocket** [comment]
Acredito que isso é normal na nossa área, eu concluí a faculdade faz 2 anos e hoje trabalho como Front-end, e me identifico com o seu caso. Apesar de gostar de todas as áreas sempre meu forte foi trabalhar com experiência do usuário, por isso sempre me encantei com Front-end. 
Igual o @lddv comentou você compreendendo o back-end vai te ajudar demais no dia a dia, principalmente pra conversar com o seu time no geral. Todo conhecimento é válido e ninguém tira de você.
Eu mesmo gostando muito de Front, também gosto de entender o back-end por que até pra fazer meus próprios projetos eu preciso entender, nem que seja o básico pra fazer algo e não ter interdependência. O que eu aprendi e continuo aplicando é não ter pressa, é ir caminhando no seu ritmo e se permitir ter várias visões e não se limitar a só uma coisa, acredito que isso vai te ajudar também nas suas soft skills. :)

Foco e paciência que as coisas vão se ajustando! :D

**Comment by ghost** [comment]
Tem toda razão @renedocket 😊 Obrigada pelo feedback, a opinião de vocês me ajuda muito!
